### PR TITLE
fix(sdk-coin-etc): Unsigned sweep error message

### PR DESCRIPTION
**Pull request by sampras-saha**
Previously error key was not read from the rpc response object,
resulting in showing undefined to the user.

Now it is properly reading **error** key from the response object and
displaying proper error message to the user.

TICKET: COIN-2221

### CODE REVIEW

**Review by mukeshsp — DISMISSED** [review]

File: modules/utxo-bin/package.json [hunk 1]
```diff
@@ -29,7 +29,7 @@
     "@bitgo/blockapis": "^1.10.6",
     "@bitgo/statics": "^50.8.0",
     "@bitgo/utxo-lib": "^11.0.1",
-    "@bitgo/unspents": "^0.47.13",
+    "@bitgo/unspents": "^0.47.14",
     "@bitgo/wasm-miniscript": "^1.8.0",
```
Comment (mukeshsp) [inline modules/utxo-bin/package.json:33]: Please remove conflict

File: modules/utxo-bin/package.json [hunk 2]
```diff
@@ -29,7 +29,7 @@
     "@bitgo/blockapis": "^1.10.6",
     "@bitgo/statics": "^50.8.0",
     "@bitgo/utxo-lib": "^11.0.1",
-    "@bitgo/unspents": "^0.47.13",
+    "@bitgo/unspents": "^0.47.14",
```
Comment (venkateshv1266) [inline modules/utxo-bin/package.json:32]: If you are updating the dependency, please add `package-lock.json` file as well.

Comment (sampras-saha) [reply modules/utxo-bin/package.json:32]: As discussed with @venkateshv1266, the `package-lock.json` is not required as it is an [untracked file](https://github.com/BitGo/BitGoJS/blob/1fb05dfecd51bc2916fa2d13111afde81ed89935/.gitignore)

File: modules/utxo-bin/package.json (hunk 1 above)
Comment (sampras-saha) [reply modules/utxo-bin/package.json]: Resolved

**Review by venkateshv1266 — APPROVED** [review]

**Pull request merged**